### PR TITLE
Feature/use header title

### PR DESCRIPTION
**Pull request by dsnider**
Addresses #98

### CODE REVIEW

File: lib/talon/resource.ex [hunk 1]
```diff
@@ -226,6 +218,11 @@ defmodule Talon.Resource do
         Talon.Resource.name_field @__module__
       end
 
+      @spec resource_title(Module.t) :: String.t
+      def resource_title(resource) do
+        resource |> Map.get(Talon.Resource.name_field(resource.__struct__))
```
Comment (smpallen99) [inline lib/talon/resource.ex]: This does not conform to the style guide. It should be replaced with `Map.get(resource, Talon.Resource...)`

File: mix.exs [hunk 2]
```diff
@@ -25,20 +38,39 @@ defmodule Talon.Mixfile do
...
+  end
+
+  # From Mix.Phoenix
+  # This doesn't work when pulling Talon into an umbrella using a symbolic link
+  # TODO: remove
+  def in_umbrella?(app_path) do
+    umbrella = Path.expand(Path.join [app_path, "..", ".."])
+    mix_path = Path.join(umbrella, "mix.exs")
+    apps_path = Path.join(umbrella, "apps")
+    IO.inspect apps_path, label: "apps_path"
```
Comment (smpallen99) [inline mix.exs]: The inspect should be removed before merging.

**Review by smpallen99 — APPROVED** [review]
Couple minor comments

File: mix.exs (hunk 2 above)
Comment (dsnider) [reply mix.exs]: Per the TODO, I'll remove this function altogether.

**Comment by dsnider** [comment]
closes #98

**Pull request merged**